skrifa/src/outline/autohint/latin/widths.rs

```diff
@@ -134,6 +134,32 @@ mod tests {
         );
     }
 
+    #[test]
+    fn cjk_computed_widths() {
+        check_widths(
+            font_test_data::NOTOSERIFTC_AUTOHINT_METRICS,
+            super::ScriptClass::HANI,
+            [
```

Comment:
suggest comment explaining where the expected value scome from. FT I assume, but how do you actually create the data? Or is it just what we observed our code to produce?

Reply:
Noted that these values are scraped from a FT debugging session.

In this case, I just copied the test skeleton and manually transcribed the values from the debugger. I figured the odds of me making a transcription error that matches a bug are vanishingly low :) For tests with more data (segments, edges, etc), I have some ugly printf code in my local FT copy that prints the expected values to the console and I copy that into the tests. This is really handy in cases where FT uses pointers but we use indices.

I tend to only do "regression style" tests (copying in values that we produce) when we already have known good tests to start with.